### PR TITLE
Updates to cloud getting started

### PR DESCRIPTION
**Pull request by stevenGravy**
Updates:
- Shows session replays for SSH
- Expands instructions
- Shows multiple install types for all the desktops supported by Teleport
- Include Teleport Connect in next steps

### CODE REVIEW

File: docs/pages/choose-an-edition/teleport-cloud/getting-started.mdx [hunk 1]
```diff
@@ -28,19 +28,64 @@ Teleport Agent Installed
...
+
+Exit the session with `exit` and you can replay the session within Session Recordings
+![Replay Session](../../../img/cloud/online-replay-5.png)
+
 ## Step 5/5 Access from Command Line
 
 Install client libraries:
 
 <Tabs>
+  <TabItem label="MacOS">
```
Comment (zmb3) [inline docs/pages/choose-an-edition/teleport-cloud/getting-started.mdx:45]: This should probably be a partial.

File: docs/pages/choose-an-edition/teleport-cloud/getting-started.mdx [hunk 2]
```diff
@@ -28,19 +28,64 @@ Teleport Agent Installed
...
+
+  You can download one of the following .pkg installers for macOS:
+
+  |Link|Binaries|
+  |-|-|
+  |[`teleport-(=teleport.version=).pkg`](https://get.gravitational.com/teleport-ent-(=teleport.version=).pkg)|`teleport`<br/>`tctl`<br/>`tsh`<br/>`tbot`|
```
Comment (zmb3) [inline docs/pages/choose-an-edition/teleport-cloud/getting-started.mdx:51]: Let's use cdn.teleport.dev going forward.

File: docs/pages/choose-an-edition/teleport-cloud/getting-started.mdx [hunk 3]
```diff
@@ -70,6 +115,7 @@ Type exit to end this session.  Happy Teleporting!
 
 ## Next Steps
 
+- Install and use [Teleport Connect](../connect-your-client/teleport-connect.mdx) as a Desktop Graphical Client to access your resources.
```
Comment (zmb3) [inline docs/pages/choose-an-edition/teleport-cloud/getting-started.mdx:118]:
```suggestion
- Consider using our desktop app - [Teleport Connect](../connect-your-client/teleport-connect.mdx) to access your resources.
```

File: docs/pages/choose-an-edition/teleport-cloud/getting-started.mdx (hunk 1 above)
Comment (stevenGravy) [reply docs/pages/choose-an-edition/teleport-cloud/getting-started.mdx:45]: @zmb3 moved to partial and updated to use cdns

**Review by alexfornuto — APPROVED** [review]
👍 with pending edits.

**Review by zmb3 — APPROVED** [review]

**Pull request merged**